### ISSUE TITLE
Content Admins do not have correct permissions to merge tags

### ISSUE THREAD

**Issue by blms** [Princeton-CDH/geniza#1378]
## Testing notes

- [x] In the [QA site admin](https://test-geniza.cdh.princeton.edu/admin/), navigate to "Groups" and confirm that the "Content Admin" group has all of the below permissions:
  - "corpus | document | Can change document"
  - "taggit | tag | Can change tag"
  - "taggit | tag | Can delete tag"
  - "taggit | tagged item | Can add tagged item"
  - "taggit | tagged item | Can change tagged item"

---

Results in a 403 error on an attempted tag merge.

```py
permission_required = (
    "corpus.change_document",
    "taggit.change_tag",
    "taggit.delete_tag",
    "taggit.change_tagged_item",
    "taggit.add_tagged_item",
)
```

**Comment by kseniaryzhova** [comment]
@blms all permission are there, thanks! Closing!

**Comment by blms** [comment]
Reopening—This is still not working as it continues to raise a 403 despite the group having the correct permissions.

**Comment by blms** [comment]
Closing, confirmed fixed in production. Fixed by #1395
